codecov.yml

```diff
@@ -3,5 +3,8 @@ codecov:
 ignore:
   - "contracts/test/"
   - "contracts/src/mock/"
-  - "contracts/src/libraries/"
-  - "contracts/src/state/"
\ No newline at end of file
+  - "contracts/src/state/operatorsRegistry/Operators.1.sol"
+  - "contracts/src/state/river/CLValidatorCount.sol"
+  - "contracts/src/state/river/CLValidatorTotalBalance.sol"
+  - "contracts/src/state/river/LastOracleRoundId.sol"
+  - "contracts/src/state/shared/Version.sol"
```

Comment:
I am adding your note @iamsahu to the PR, so that we keep your thoughts next to the code: 

The following files were excluded from the coverage report:
- [Operators.1.sol](https://github.com/liquid-collective/liquid-collective-protocol/blob/main/contracts/src/state/operatorsRegistry/Operators.1.sol)
- [CLValidatorCount.sol](https://github.com/liquid-collective/liquid-collective-protocol/blob/main/contracts/src/state/river/CLValidatorCount.sol)
- [CLValidatorTotalBalance.sol](https://github.com/liquid-collective/liquid-collective-protocol/blob/main/contracts/src/state/river/CLValidatorTotalBalance.sol)
- [LastOracleRoundId.sol](https://github.com/liquid-collective/liquid-collective-protocol/blob/main/contracts/src/state/river/LastOracleRoundId.sol)
- [Version.sol](https://github.com/liquid-collective/liquid-collective-protocol/blob/main/contracts/src/state/shared/Version.sol)

The underlying functions in the files 2-5 are wrappers using functions from the library [LibUnstructuredStorage](https://github.com/liquid-collective/liquid-collective-protocol/blob/main/contracts/src/libraries/LibUnstructuredStorage.sol). The library has 100% coverage. So, it is deemed safe to remove it from coverage report.

The file 1 defines CRUD operations for storing operators. However, this was made redundant when the new structure was introduced in Operators.2.sol. So, the code in the file is no longer used and is present in the repository as it is being used in the [OperatorsRegistry](https://github.com/liquid-collective/liquid-collective-protocol/blob/main/contracts/src/OperatorsRegistry.1.sol) for migration. The migration process has been executed and there is no way to retrigger the process in the currently deployed contract. So, it is deemed safe to not have this file in the coverage report.

Reply:
Left unresolved for future visibility